PS.g4

```diff
@@ -319,7 +364,7 @@ args: (expr ',' args) | expr;
 
 limit_sub:
     UNDERSCORE L_BRACE
-    (LETTER | SYMBOL)
+    (LETTER | GREEK_LETTER)
```

Comment:
@tragessere should this also include `SYMBOL` still? is there a reason why pi and infty couldn't be used in a limit?

Reply:
@builtbywill This is the left hand side of the limit so it would be saying that the value of pi/infinity are approaching some number/value